### PR TITLE
Proposed minimally viable CNCT website

### PR DESCRIPTION
**Pull request by iredwards**
The files in this PR are for an update for the CNCT Github Pages site: http://samsung-cnct.github.io/  

They can be previewed on the staging site: https://samsung-cnct.github.io/docs-ghp-staging/index.html

I am highly confident that the contents of this PR fall short of Perfection in some respects.  

But! I believe they are better than what is up now. Your suggestions, gripes, and praise will be heartily welcomed. Especially raise a flag if you have a compelling reason why this content should not be published now while I work on improving it.  

under construction

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,37 +1,256 @@
...
+
+<div class="post-content">
+
+   
+
+    
+
+  <p>Samsung SDS provides frictionless software solutions to help you optimize productivity, make the right decisions, and strengthen your organization. The Samsung SDS Cloud Native Computing Team (CNCT) was created to advance this mission through research into practices that improve the velocity and efficiency of DevOps teams, within Samsung, for its clients and partners, and for the community.</p>
+
+<p>As a technology team committed to invisible infrastructure, we bring insight into the requirements of modern web services delivered at scale. Samsung SDS has been a driving force behind Kubernetes and is committed to advancing containerized architectures. This work benefits Samsung’s own deployments and execution in the hybrid cloud environment through improved automation, flexibility, co-tenancy, and efficiency.</p>
```
Comment (spiffxp) [inline index.html]: "invisible infrastructure" [citation needed]

Comment (spiffxp) [inline index.html]: s/bring/provide

File: index.html [hunk 2]
```diff
@@ -1,37 +1,256 @@
...
+
+
+
+<div class="post-content">
+
+   
+
+    
+
+  <p>Samsung SDS provides frictionless software solutions to help you optimize productivity, make the right decisions, and strengthen your organization. The Samsung SDS Cloud Native Computing Team (CNCT) was created to advance this mission through research into practices that improve the velocity and efficiency of DevOps teams, within Samsung, for its clients and partners, and for the community.</p>
```
Comment (spiffxp) [inline index.html]: IMO we improve the efficiency of more teams than just DevOps, I would propose dropping the qualifier

File: index.html [hunk 3]
```diff
@@ -1,37 +1,256 @@
...
+
+<p>As a technology team committed to invisible infrastructure, we bring insight into the requirements of modern web services delivered at scale. Samsung SDS has been a driving force behind Kubernetes and is committed to advancing containerized architectures. This work benefits Samsung’s own deployments and execution in the hybrid cloud environment through improved automation, flexibility, co-tenancy, and efficiency.</p>
+
+<p>Samsung SDS is committed to active participation in a vibrant open-source community that delivers production-grade infrastructure software. The CNCT contributes tools and research to advance cloud native innovation. These tools are designed to help you scale your infrastructure no matter where your services are deployed.</p>
```
Comment (spiffxp) [inline index.html]: we haven't talked to "you" yet in this doc, I'd prefer we drop it here for consistency... "scale services regardless of where the underlying infrastructure is located"

File: index.html [hunk 4]
```diff
@@ -1,37 +1,256 @@
...
+
+<p>The CNCT also provides consulting to organizations making the shift to Cloud Native computing. Services include guidance refactoring development and operations groups, working with development teams to adopt technologies like Kubernetes, and performing SRE and (virtual) cluster operations services.</p>
```
Comment (spiffxp) [inline index.html]: what does virtual mean in this context

File: k2-helm-charts.html [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+
+   
+
+    
+
+  <p>CNCT creates Helm charts that can be useful for deploying a K2 cluster with the functionality your application requires.</p>
+
+<p>For an introduction to Helm and how to use charts in your K2 application: <a href="https://helm.sh/">helm.sh</a></p>
+
+<p>These charts are automatically linted and published with Jenkins CI to the CNCT Helm repo: <a href="http://atlas.cnct.io/">atlas.cnct.io</a>. Source for these charts is available on GitHub at https://github.com/samsung-cnct/k2-charts.</p>
```
Comment (spiffxp) [inline k2-helm-charts.html]: link to k2-charts

File: cnct_tenets.html [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+    <a href="#">About CNCT</a>
+    <ul>
+      
+      
+      <li><a href="index.html">Samsung SDS CNCT Home</a></li>
+      
+      
+      
+      
+        <li class="active"><a href="cnct_tenets.html">CNCT Tenets</a></li>
```
Comment (alejandroEsc) [inline cnct_tenets.html]: spacing on this second item is consistently, across other html files, inconsistently spaced with the the previous submenu object. nit.

File: cnct_tenets.html [hunk 7]
```diff
@@ -0,0 +1,271 @@
...
+    </ul>
+    
+  <li>
+    <a href="#">CNCT K2 Projects</a>
+    <ul>
+      
+      
+      <li><a href="aboutK2.html">About K2</a></li>
+      
+
```
Comment (alejandroEsc) [inline cnct_tenets.html]: do we need so many spaces between submenu objects?

**Comment by alejandroEsc** [comment]
So i see 3 additional concerns:
1. When clicking through the menu items the positioning for the menu bar keeps changing. Would be nice to keep this static.
2. Would be nice if submenu options would be indented relative to menu option. I can tell that the size changes, but slightly and hard to really understand, specially when choosing collapsing it will definitely help keep track
3. Why is the video submenu highlighted dark gray?

File: about.html [hunk 8]
```diff
@@ -0,0 +1,273 @@
...
+})
+  
+});
+</script>
+
+<div id="toc"></div>
+
+    
+
+  <p>The Cloud Native Compute Team is based in the historic Smith Tower in Seattle’s Pioneer Square.</p>
```
Comment (iredwards) [inline about.html]: That file should not have been built. I didn't finish it and didn't add it to the ToC, which is why you couldn't navigate to it. The html won't be built now until it is ready.

File: aboutK2.html [hunk 9]
```diff
@@ -0,0 +1,274 @@
...
+
+
+
+<div class="post-content">
+
+   
+
+    
+
+  <p><a href="https://github.com/samsung-cnct/k2">K2</a> is an orchestration and cluster level management system for <a href="https://kubernetes.io">Kubernetes</a>. K2 will create a production scale Kubernetes cluster on a range of platforms using its default settings. This can be especially useful if you are getting
```
Comment (iredwards) [inline aboutK2.html]: Suggested changes made

File: aboutK2.html [hunk 10]
```diff
@@ -0,0 +1,274 @@
...
+started and don’t need a HA production level cluster right away. When you are ready to optimize your cluster for your own environment and use case, K2 provides a rich set of configurable options.</p>
+
+<p>We (Samsung CNCT) built this tool to aid in our own research into high performance and reliability for the Kubernetes control plane. We realized this would be a useful tool for the public at large and released it as <a href="https://github.com/samsung-cnct/kraken">Kraken</a>. Kraken was great but it was developed quickly for research. After using it ourselves for almost a year and identifying some pain points we decided it was best to build anew, bringing the best parts forward. It continues to use Ansible and Terraform because we believe those tools provide flexible and powerful abstractions at the right layers.</p>
+
+<p>K2 provides the same functionality with much cleaner internal abstractions. This makes it easier for both external and internal contributions. It will also allow us to continue to quickly improve and evolve with the Kubernetes ecosystem as a whole.</p>
```
Comment (iredwards) [inline aboutK2.html]: Suggested changes made.  Last 2 sentences sentenced to deletion.

File: aboutK2.html [hunk 11]
```diff
@@ -0,0 +1,274 @@
...
+
+<h2 id="what-is-k2-for">What is K2 for</h2>
+<p>K2 is targeted at operations teams that need to support Kubernetes, a practice becoming known as ClusterOps. K2 provides a single interface where you can manage your Kubernetes clusters across all environments.</p>
```
Comment (iredwards) [inline aboutK2.html]: Suggested changes made.

File: aboutK2.html [hunk 12]
```diff
@@ -0,0 +1,274 @@
...
+
+<p>K2 uses a single file to drive cluster configuration. This makes it easy to check the configuration file into a VCS of your choice and solve two major problems:</p>
+
+<ol>
+  <li>
+    <p>Keep your cluster configuration under version control as you promote changes from dev through to production, for either existing cluster configurations or brand new ones</p>
```
Comment (iredwards) [inline aboutK2.html]: Now:

K2 uses a single file to drive cluster configuration, providing two major benefits:

- It simplifies keeping your cluster configuration under version control as you promote changes from dev through to production. This applies to existing cluster configurations and brand new ones.

- It enables Continuous Integration for applications running on sandboxed and transient Kubernetes clusters.

We believe this simplified configuration management facility is essential for effectively and efficiently operating a Kubernetes based infrastructure.

File: aboutK2.html [hunk 13]
```diff
@@ -0,0 +1,274 @@
...
+  </li>
+  <li>
+    <p>Enable Continuous Integration for developer applications against sandboxed and transient Kubernetes clusters. K2 provides a destroy command that will clean up all traces of the temporary infrastructure.</p>
```
Comment (iredwards) [inline aboutK2.html]: see above

File: aboutK2.html [hunk 14]
```diff
@@ -0,0 +1,274 @@
...
+  </li>
+</ol>
+
+<p>We believe solving these two problems is a baseline for effectively and efficiently nurturing a Kubernetes based infrastructure.</p>
```
Comment (iredwards) [inline aboutK2.html]: see above

File: aboutK2.html [hunk 15]
```diff
@@ -0,0 +1,274 @@
...
+
+<h2 id="k2-support-projects">K2 Support Projects</h2>
+<p>K2 has supporting tools, including a command-line interface wrapper, Helm charts, and logging tools. These tools are available on GitHub.</p>
```
Comment (iredwards) [inline aboutK2.html]: Now:
K2 is supported by a variety of other tools, including:

- [k2cli](k2cli) - a command-line interface wrapper
- [k2-charts](k2-helm-charts) - Helm charts for a variety of useful services
- [logging tools](k2-logging) - tools for centralized logging

These tools are available on GitHub.

File: index.html (hunk 2 above)
Comment (iredwards) [inline index.html]: done

File: index.html (hunk 1 above)
Comment (iredwards) [inline index.html]: quote marks added, but no obvious citation found on google search.  Wikipedia has page for "99% Invisible".  So close...

File: index.html (hunk 3 above)
Comment (iredwards) [inline index.html]: done

File: index.html (hunk 4 above)
Comment (iredwards) [inline index.html]: I wondered about that too. Not sure what SRE is either.

File: k2-helm-charts.html (hunk 5 above)
Comment (iredwards) [inline k2-helm-charts.html]: This is on the charts page already.  I removed this line, and turned the first mention of Helm into a link to the Helm site.

File: cnct_tenets.html (hunk 7 above)
Comment (iredwards) [inline cnct_tenets.html]: Yeah, the ToC control is confusing especially with so few pages at this point.  It makes more sense for a bigger doc set, such as this one (where I copied it from):
http://idratherbewriting.com/documentation-theme-jekyll/index.html

There are other nav control solutions that are better for smaller sites. Like blogs. But those have problems for documentation sites. I'm not sure just yet whether to re--skin the tre control, use a different one, or add more content.  Option 3 is the obvious choice, but requires more effort.

File: index.html (hunk 1 above)
Comment (spiffxp) [inline index.html]: I regularly wear a tshirt I bought to support a podcast called 99% Invisible.

This thing we are committed to. What is it.

**Pull request closed without merging**